### PR TITLE
Improves Normal View on Research Console

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
I've changed the 'Normal View' On the research menu so it shows all available items on all sections.

Before:
image

After:
uAtDVaHpT3

## Why It's Good For The Game

I've been going one by one through all the research nodes and checking what items they contain manually, this makes it much easier to see what you want to research along with what you have researched. No more researching things and having them lost to the abyss.

## Changelog
:cl:
Improved Normal View for RD computer.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
You need to use a label for the changelog otherwise it wont work
:cl:
tweak: Improved normal view
/:cl:

**Comment by Axanthic** [comment]
> You need to use a label for the changelog otherwise it wont work
> 🆑
> tweak: Improved normal view
> /🆑

I don't get it what do you mean?

**Comment by Time-Green** [comment]
> > You need to use a label for the changelog otherwise it wont work
> > 🆑
> > tweak: Improved normal view
> > /🆑
> 
> I don't get it what do you mean?

There's a bunch of labels you can use when making changelogs, it's actually already filled out for you when making a new PR. A few examples are tweak, add and fix. The changelog will not work if you don't start with one of those.

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -709,7 +709,7 @@ Nothing else in the console has ID requirements.
 		display_name = "<A href='?src=[REF(src)];view_node=[node.id];back_screen=[screen]'>[display_name]</A>"
 	l += "<div class='statusDisplay technode'><b>[display_name]</b> [RDSCREEN_NOBREAK]"
 	if(minimal)
-		l += "<br>[node.description]"
+		l += ""
```
Comment (AnturK) [inline code/modules/research/rdconsole.dm:712]: If you're not adding anything here now, you should get rid of the whole if. And remove the minimal parameter if it's not used anywhere else.

**Review by AnturK — CHANGES_REQUESTED** [review]

**Comment by Misha600** [comment]
Please god yes, we need it

**Comment by trerri** [comment]
I fear the poopy HTML UI will absolutely shit its pants with this many icons. It already was the cause of a really bad bug that made all UI sluggish that even reconnects couldn't fix.

**Pull request closed without merging**